python/cuml/decomposition/pca.pyx

```diff
@@ -244,15 +244,15 @@ class PCA(Base):
     ----------
     components_ : array
         The top K components (VT.T[:,:n_components]) in U, S, VT = svd(X)
-    explained_variance_ : array
+    _explained_variance_ : array
```

Comment:
sklearn doesn’t put underscores before these attributes. Was this done intentionally?

Reply:
that's why I get for using find and replace indiscriminately